raiden_contracts/constants.py

```diff
@@ -11,6 +13,57 @@
 CONTRACT_MONITORING_SERVICE = 'MonitoringService'
 CONTRACT_RAIDEN_SERVICE_BUNDLE = 'RaidenServiceBundle'
 
+# Deployed contract information
+# Deployed to Ropsten revival on 2018-09-03 from
+# raiden-contracts@fc1c79329a165c738fc55c3505cf801cc79872e4
+ROPSTEN_REGISTRY_ADDRESS = '0xf2a175A52Bd3c815eD7500c765bA19652AB89B30'
+ROPSTEN_DISCOVERY_ADDRESS = '0xEEADDC1667B6EBc7784721B123a6F669B69Eb9bD'
+ROPSTEN_SECRET_REGISTRY_ADDRESS = '0x16a25511A92C5ebfc6C30ad98F754e4c820c6822'
+# Deployed to Ropsten revival on 2018-09-21 from
+# raiden-contracts@bfb24fed3ebda2799e4d11ad1bb5a6de116bd12d
+ROPSTEN_LIMITS_REGISTRY_ADDRESS = '0x6cC27CBF184B4177CD3c5D1a39a875aD07345eEb'
+ROPSTEN_LIMITS_DISCOVERY_ADDRESS = '0xcF47EDF0D951c862ED9825F47075c15BEAf5Db1B'
+ROPSTEN_LIMITS_SECRET_REGISTRY_ADDRESS = '0x8167a262Fa3Be92F05420675c3b409c64Be3d348'
+
+
+# Network configurations
+START_QUERY_BLOCK_KEY = 'DefaultStartBlock'
+
+
+class NetworkType(Enum):
+    MAIN = 1
+    TEST = 2
+
+
+ID_TO_NETWORK_CONFIG = {
+    3: {
```

Comment:
Can we have an additional `Enum` with the chain id?
E.g.
```
class ChainId(Enum):
    ROPSTEN = 3
    MAINNET = 1
   ...
```

Reply:
Yep!

Reply:
Ah, I was suggesting to have `ChainId. ROPSTEN` instead of `3` here.

Reply:
Yes we should. I missed it during the change. Let me add it now.